autobahn/asyncio/component.py

```diff
@@ -289,7 +289,7 @@ def _connect_transport(self, loop, transport, session_factory):
             assert(False), 'should not arrive here'
 
     # async function
-    def start(self, loop=None):
+    def start(self, loop=None, log_level=None):
         """
```

Comment:
I actually don't think this belongs here (nor the start_logging call) -- it is already done in the ``run`` method and calling ``start`` on a component is (at least intended as) the lower-level API. That is, if you're already controlling your own event-loop. In this case, I would presume most users would also be controlling the log-starting themselves, too.

The idea is that "most" users should just call ``run([component0, component1])`` but if you wanted more control you'd call start yourself and deal with the deferreds/futures -- adding one more line in this case for ``start_logging`` yourself seems reasonable (since you'd also have to arrange for starting the event-loop anyway).

Reply:
Yep I came to the same conclusion thinking about this today - if you're managing the loop you're also managing the logging so I will remove this.

Reply:
Bump; looks like `log_level=` is still in this diff?